### PR TITLE
Added A3! Wiki presence

### PR DESCRIPTION
**Pull request by riarisu**
Version 1.0.0: view main page, view page, view history, edit page

image
image
image
image

### CODE REVIEW

File: websites/A/A3! Wiki/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "A3! Wiki",
+    "author": {
+      "name": "Riarisu",
+      "id": "506090262964076555"
+    },
+    "category": "anime",
+    "logo": "https://yaycupcake.com/a3/skins/common/images/wiki.png",
```
Comment (Bas950) [inline websites/A/A3! Wiki/dist/metadata.json:9]: upload to imgur

File: websites/A/A3! Wiki/presence.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({
+  clientId: "837305361639538728"
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
+let title;
+const actionURL = new URL(document.location.href);
+const title2URL = new URL(document.location.href);
```
Comment (Bas950) [inline websites/A/A3! Wiki/presence.ts:8]: merge consts

File: websites/A/A3! Wiki/presence.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+presence.on("UpdateData", () => {
+
+  const presenceData: PresenceData = {
+    largeImageKey: "wiki"
+  };
+  presenceData.startTimestamp = browsingStamp;
+
+  title = document.querySelector(".firstHeading");
+  const actionResult = actionURL.searchParams.get("action");
+  const title2Result = title2URL.searchParams.get("title");
```
Comment (Bas950) [inline websites/A/A3! Wiki/presence.ts:19]: merge const

**Review by Bas950 — CHANGES_REQUESTED** [review]
Add some more things to the presence if you can...

File: websites/A/A3! Wiki/dist/metadata.json (hunk 1 above)
Comment (riarisu) [reply websites/A/A3! Wiki/dist/metadata.json:9]: okay! I will

File: websites/A/A3! Wiki/presence.ts (hunk 2 above)
Comment (riarisu) [reply websites/A/A3! Wiki/presence.ts:8]: ah yea I saw the code checks from yesterday but I was waiting for your reply first, ok will work for it!

**Comment by ririxidev** [comment]
Heya @riarisu!
We are so sorry but we need to deny your pull request regarding this presence due of:
 - Low quality code.
 - Site after reviewing by us, is not qualified for presence. Sorry!
 
Regards,
**ririxi**. PreMiD Reviewer Team.
Contact: mail@ririxi.dev, ririxi#2721.

**Comment by riarisu** [comment]
> 
> 
> Heya @riarisu!
> We are so sorry but we need to deny your pull request regarding this presence due of:
> 
>     * Low quality code.
> 
>     * Site after reviewing by us, is not qualified for presence. Sorry!
> 
> 
> Regards,
> **ririxi**. PreMiD Reviewer Team.
> Contact: [mail@ririxi.dev](mailto:mail@ririxi.dev), ririxi#2721.

Hello, I'm sorry I just read this. May I know what's the requirement to be added as a presence? Because I thought wikis are okay since I've seen a lot of wikis, even the Fandom/Wikia one, in the presence store. If the website isn't qualified then I might not continue to fix the code and re-submit the pull request. Thank you! I'll probably message your Discord though if this comment won't be noticed.

**Comment by ririxidev** [comment]
Yup, that's right but these wiki pages are just better in code way and implements every subpage on site. What would be a requirement? Problem is, this is likely a subpage that is only supported. If this site would have more options or for example would be only wiki, maybe we would accept it, but for now, we don't qualify this as good site for presence.

**Comment by riarisu** [comment]
Ahh I understand, thank you very much for the reply! I'll tell my wiki team that we can't add our wiki to the presence then.

**Pull request closed without merging**